recipes/enkits/all/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 2.8.12)
```

Comment:
```suggestion
cmake_minimum_required(VERSION 3.1)
```

Reply:
upstream cmakelists require 3.0. https://github.com/dougbinks/enkiTS/blob/master/CMakeLists.txt#L1

Reply:
The hooks require 3.1.
I win :stuck_out_tongue:

Reply:
Its only applies to the file test_package/CMakeLists.txt, not the wrapper file ("The CMake wrapper file can require CMake 2.8, because Conan recipe and the test package are totally separated. "). :wink: